File: translation_output/mistral/codenet/python/java-sanitized/p03042.java
import java.util.Scanner;

public class p03042 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String n = scanner.nextLine();
        int a = Integer.parseInt(n.charAt(0) + n.charAt(1));
        int b = Integer.parseInt(n.charAt(2) + n.charAt(3));
        if (b == 0 || b > 12) {
            if (1 <= a <= 12) {
                System.out.println("MMYY");
            } else {
                System.out.println("NA");
            }
        } else if (1 <= a <= 12) {
            System.out.println("AMBIGUOUS");
        } else {
            System.out.println("YYMM");
        }
    }
}

File: translation_output/no_test/mistral/codenet/java/python-sanitized/p03042.py
import sys

S = int(input())
yy = S // 100
mm = S % 100

if 1 <= yy and yy <= 12:
    if 1 <= mm and mm <= 12:
        print("AMBIGUOUS")
    else:
        print("MMYY")
else:
    if 1 <= mm and mm <= 12:
        print("YYMM")
    else:
        print("NA")